python/pyspark/sql/dataframe.py

```diff
@@ -4430,6 +4430,48 @@ def withColumnRenamed(self, existing: str, new: str) -> "DataFrame":
         """
         return DataFrame(self._jdf.withColumnRenamed(existing, new), self.sparkSession)
 
+    def withColumnsRenamed(self, *colsMap: Dict[str, Column]) -> "DataFrame":
```

Comment:
Why it is a map of str and Column? Based on `withColumnRenamed`, I think it should be a map of str and str?

Reply:
Good catch!